FlashSpace/Features/Workspaces/WorkspaceHotKeys.swift

```diff
@@ -41,10 +41,17 @@ final class WorkspaceHotKeys {
         guard let shortcut = workspace.activateShortcut else { return nil }
 
         let action = { [weak self] in
-            guard let updatedWorkspace = self?.workspaceRepository.workspaces
+            guard let self, let updatedWorkspace = self.workspaceRepository.workspaces
                 .first(where: { $0.id == workspace.id }) else { return }
 
-            self?.workspaceManager.activateWorkspace(updatedWorkspace, setFocus: true)
+            if self.workspaceSettings.enableWorkspaceTransition {
+                WorkspaceTransitionManager.shared.enableTransitionEffects = true
+                WorkspaceTransitionManager.shared.performTransition(direction: .right) {
+                    self.workspaceManager.activateWorkspace(updatedWorkspace, setFocus: true)
+                }
+            } else {
+                self.workspaceManager.activateWorkspace(updatedWorkspace, setFocus: true)
+            }
```

Comment:
Could be simplified:
1. Remove `enableTransitionEffects`
2. Access settings within `WorkspaceTransitionManager`: 
    ```swift
     private var isEnabled: Bool { AppDependencies.shared.workspaceSettings }
    ```
3. Replace it here and in other places in a similar way:
    ```swift
     WorkspaceTransitionManager.shared.performTransition(direction: .right) {
          self.workspaceManager.activateWorkspace(updatedWorkspace, setFocus: true)
      }
     ```